diff --git a/node_modules/sigma/dist/colors-beb06eb2.esm.js b/node_modules/sigma/dist/colors-beb06eb2.esm.js
index b7130d1..bf101b6 100644
--- a/node_modules/sigma/dist/colors-beb06eb2.esm.js
+++ b/node_modules/sigma/dist/colors-beb06eb2.esm.js
@@ -51,7 +51,6 @@ function _nonIterableRest() {
 function _slicedToArray(r, e) {
   return _arrayWithHoles(r) || _iterableToArrayLimit(r, e) || _unsupportedIterableToArray(r, e) || _nonIterableRest();
 }
-
 var HTML_COLORS = {
   black: "#000000",
   silver: "#C0C0C0",
@@ -267,7 +266,10 @@ for (var htmlColor in HTML_COLORS) {
   FLOAT_COLOR_CACHE[HTML_COLORS[htmlColor]] = FLOAT_COLOR_CACHE[htmlColor];
 }
 function rgbaToFloat(r, g, b, a, masking) {
-  INT32[0] = a << 24 | b << 16 | g << 8 | r;
+  const r_= Math.floor(r * a / 255)
+  const g_= Math.floor(g * a / 255)
+  const b_= Math.floor(b * a / 255)
+  INT32[0] = a << 24 | b_ << 16 | g_ << 8 | r_;
   if (masking) INT32[0] = INT32[0] & 0xfeffffff;
   return FLOAT32[0];
 }
